extensions/binding/org.eclipse.smarthome.binding.digitalstrom/ESH-INF/i18n/digitalstrom_de.properties

```diff
@@ -0,0 +1,128 @@
+#group-parms
+DSS_BRIDGE_PARM_GROUP_CONNECTION_LABEL = Verbindungs Konfigurationen
+DSS_BRIDGE_PARM_GROUP_CONNECTION_DESC = Einstellungen der Verbindungs Konfigurationen, um eine Verbindung zum digitalSTROM-Server herzustellen.
+
+DSS_BRIDGE_PARM_GROUP_GENERAL_LABEL = Gennerelle Konfigurationen
+DSS_BRIDGE_PARM_GROUP_GENERAL_DESC = Hier k�nnen Sie gennerelle Einstellungen f�r das Binding einstellen.
+
+DSS_BRIDGE_PARM_GROUP_INFO_LABEL = digitalSTROM-Server Informationen
+DSS_BRIDGE_PARM_GROUP_INFO_DESC = Zeigt Informationen zum digitalSTROM-Server.
+
+#parameters
+DSS_BRIDGE_PARM_APP_TOKEN_LABEL = Application-Token
+DSS_BRIDGE_PARM_APP_TOKEN_DESC = Der Application-Token zu Authentifizierung am digitalSTROM-Server .
+
+DSS_BRIDGE_PARM_IP_ADRESS_LABEL = Netzwerkadresse
+DSS_BRIDGE_PARM_IP_ADRESS_DESC = Netzwerkadresse des digitalSTROM-Server.
+
+DSS_BRIDGE_PARM_USERNAME_LABEL = Benutzername
+DSS_BRIDGE_PARM_USERNAME_DESC = Regestierter Benutzername um sich beim digitalSTROM-Server zu autentifizieren.
+
+DSS_BRIDGE_PARM_PASSWORD_LABEL = Passwort
+DSS_BRIDGE_PARM_PASSWORD_DESC = Regestiertes Passwort zum Benutzername, um sich beim digitalSTROM-Server zu autentifizieren.
+
+DSS_BRIDGE_PARM_DSID_LABEL = digitalSTROM-Sever ID
+DSS_BRIDGE_PARM_DSID_DESC = Die eindeutige Kennung des DigitalSTOM-Severs.
+
+DSS_BRIDGE_PARM_DS_NAME_LABEL = Name der digitalSTROM-Installation
+DSS_BRIDGE_PARM_DS_NAME_DESC = Zeigt den Benutzerdefinierten Namen der digitalSTROM-installation.
+
+DSS_BRIDGE_PARM_SENSOR_INTER_LABEL = Aktualisierungsintervall der Sensorwerte
+DSS_BRIDGE_PARM_SENSOR_INTER_DESC = Stellt den Aktuallisierungsintervall in Sekunden ein, in dem die Ger�te Sensorwerte, welche eine h�here Priorit�t als 'never' haben, aktualisiert werden.
+
+DSS_BRIDGE_PARM_TOTAL_POWER_INTER_LABEL = Aktualisierungsintervall der gesamt Stromverbr�uche 
+DSS_BRIDGE_PARM_TOTAL_POWER_INTER_DESC = Stellt den Aktuallisierungsintervall in Sekunden ein, in dem der gesamt Stromverbrauch und gesamt Stromz�hlerwert von digitalSTROM aktualisiert werden.
+
+DSS_BRIDGE_PARM_SENSOR_WAIT_LABEL = Wartezeit Sensorauswertung
+DSS_BRIDGE_PARM_SENSOR_WAIT_DESC = Wartezeit zwischen der Auswertung der Sensorwerte, sowie die Auslesung der Szenen in Sekunden. <b>ACHTUNG:<b> DigitalSTORM Regel 8 und 9 fordern eine Wartezeit von 1er Minute. Werte unter 60 Sekunden k�nnten das digitalSTROM-System beeintr�chtigen.
+
+DSS_BRIDGE_PARM_TRASH_DELEATE_LABEL = Tage nachdem nicht ereichbare digitalSTROM-Ger�te gel�scht werden
+DSS_BRIDGE_PARM_TRASH_DELEATE_DESC = Tage nachdem die tempor�r gespeicherten digitalSTROM Ger�tekonfiguration von nicht ereichbare digitalSTROM-Ger�te endg�ltig gel�scht werden.
+
+DSS_BRIDGE_PARM_CERT_PARTH_LABEL = Pfad zum SLL Zertifikat
+DSS_BRIDGE_PARM_CERT_PARTH_DESC = Hier k�nnen sie den Pfad zum SLL-Zertifikat f�r den digitalSTROM-Server angeben. Sie k�nnen es vom digitalSTROM-Server herunterladen. Andernfalls wird das SSL-Zertifikat ignoriert.
```

Comment:
typo: "SSL"
What kind of path is this? An absolute path of a local folder?
This might not be very convenient. Could there be a certificate pinning instead? Accept it once during setup (and automatically store it locally) and afterwards don't accept any other certificate?